File: Presentation/Fintrak.Presentation.WebClient/app/core/models/account/account-login-model.js
window.Fintrak = {};
(function (se) {
    var rootPath;
    se.rootPath = rootPath;
}(window.Fintrak));
Fintrak.rootPath = '/';

(function (ft) {
    var AccountLoginModel = function () {

        var self = this;

        self.CompanyCode = '';
        self.LoginID = '';
        self.Password = '';
        self.RememberMe = false;
    }
    ft.AccountLoginModel = AccountLoginModel;
}(window.Fintrak));

File: Presentation/Fintrak.Presentation.WebClient/Models/Core/AccountLoginModel.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Fintrak.Presentation.WebClient.Models
{
    public class AccountLoginModel

    {
        public string CompanyCode { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string LoginID { get; set; }
        public string Password { get; set; }
        public bool RememberMe { get; set; }
        public string ReturnUrl { get; set; }
    }

}